docs/source/integrations/mobile/ios-fairplay-sdk.md

```diff
@@ -87,125 +86,115 @@ More information about ATS can be found in Apple’s TechNote:
 
 ## Preparation
 
-Before you can use the demo project or framework in your own project, for each instance you will require a URL to correctly prepared content, a content ID for the content (also referred to as assetID in previous versions), and a VUDRM token which must be correctly generated for the type of instance you wish to create. Example VUDRM token type templates available are:
+The demo project is based on Apples FairPlay SDK example application. This is configured using the projects `Streams.plist` file. 
 
-- Fairplay Rental `{"type": "r","duration_rental": 3600}`
- - Fairplay Rental tokens may only be used to stream online content to devices running iOS 9.x or higher.
-- Fairplay Persist `{"type": "p","duration_persist": 3600}`
- - Fairplay Persist tokens may be used to stream online content and play offline (downloaded) content to devices running iOS 10.x or higher.
+To provide complete configuration for each `Stream` object in the `Streams.plist`you will require:
+
+ - An `skd://` URI entry in the `content_key_id_list`. The correct URI can be obtained by retrieving the manifest and parsing out the `URI` from the `EXT-X-SESSION-KEY` or `EXT-X-KEY`, this can be done in code or manually with curl in the Terminal app, eg enter: `curl https://eample.cloudfront.net/example-demo/examplecontent/examplecontent.ism/.m3u8`. 
```

Comment:
```suggestion
 - An `skd://` URI entry in the `content_key_id_list`. The correct URI can be obtained by retrieving the manifest and parsing out the `URI` from the `EXT-X-SESSION-KEY` or `EXT-X-KEY`, this can be done in code or manually with curl in the Terminal app, e.g. enter: `curl https://eample.cloudfront.net/example-demo/examplecontent/examplecontent.ism/.m3u8`. 
```